### DISCUSSION TITLE
re-order frame position

### DISCUSSION THREAD

**Discussion by eliramalho** [senshu/Sozi#586] [Ideas]
Hi !
First, thank you for all your effort doing and maintaining this great small program and make it free to use!
I love this to prepare some lectures for my students.

However, It would be great if the layers and frames positions could be changed by dragging.

Sometimes a position of a frame is great to be  like a "transition" between scenes/frames. Duplicating, only make it next to the other one, and I can't drag it for last position…

Thanks a lot!
Greetings from Portugal

**Comment by aumouvantsillage** [comment]
Thanks for your feedback.

You can already reorder frames, but not by dragging : select the frames to move and click on the tiny arrow buttons on the left or right of the frame title at the destination location. There is no plan to support drag-and-drop in the near future, but if someone is willing to develop this feature, I can provide guidance.

This is also one of the topics I need to address in the forthcoming [Sozi Guide](http://sozi.guide). It has been asked several times but the answer has never been added to the documentation.
